Added documentation for custom floor selector and map padding (#85)
MapsPeople/docsite: docs/01-getting-started/ios/v4/custom-map-padding.mdx

Reviewer: I think this should be moved to Map / Map Styling / Custom Map Padding

---
title: Custom Map Padding
hide_title: false
hide_table_of_contents: true
sidebar_position: 4
---

## How use map padding

> If your app needs features like a search bar, a settings menu, or other components,

Reviewer: ```suggestion
If your app uses UI components that overlays the map, like a search bar, a bottom sheet or other components,
```

>  you'll have to add padding to the MPMapControl. The first step is to create a new UIView or use an existing one.

Reviewer: ```suggestion
 you'll have to add padding to the `MPMapControl`. The first step is to create a new `UIView` or use an existing one.
```

```swift
// Creating a simple red view for the demo
let testView = UIView()
testView.translatesAutoresizingMaskIntoConstraints = false
testView.backgroundColor = .red
self.view.addSubview(testView)
```

> Add Constraints to the newly created UIView

Reviewer: ```suggestion
Add Constraints to the newly created `UIView`:
```

```swift
// Adding Auto Layout constraints for the red view
testView.leadingAnchor.constraint(equalTo: self.view.leadingAnchor).isActive = true
testView.trailingAnchor.constraint(equalTo: self.view.trailingAnchor).isActive = true
testView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor).isActive = true
testView.heightAnchor.constraint(equalToConstant: 250).isActive = true
```

> Add mapPadding to mapControl

Reviewer: ```suggestion
Add mapPadding to `mapControl`:
```
